ProjectShop/order/views.py

```diff
@@ -0,0 +1,19 @@
+from django.shortcuts import render
+from rest_framework import generics
+from order.serializers import OrderDetailSerializer
+from order.models import Order
+from order.serializers import OrderListSerializer
+from rest_framework.permissions import IsAuthenticated, IsAdminUser
```

Comment:
According to PEP 8:
Imports should be grouped in the following order:
1. Standard library imports.
2. Related third-party imports.
3. Local application/library-specific imports.
You should put a blank line between each group of imports.